src/main/connector/system/Windows/Windows.yaml

```diff
@@ -267,8 +256,8 @@ monitors:
         attributes:
           id: $1
         metrics:
-          system.disk.io{disk.io.direction="read"}: $2
-          system.disk.io{disk.io.direction="write"}: $3
+          system.disk.operations.bytes{direction="read"}: $2

```

Comment:
I'm sorry but this is breaking compatibility with [OpenTelemetry semantic conventions](https://opentelemetry.io/docs/specs/semconv/system/system-metrics/#metric-systemdiskio)

Please keep `system.disk.io`. Similarly, other changes in metric names are breaking compatibility and must be fixed before we can merge.

Reply:
@bertysentry thanks for the review.
If we keep the metric `system.disk.io`, is it possible to have bytes as the unit?
Or must the metric necessarily be in IO?